docs/INSTALL.md

```diff
@@ -188,7 +186,7 @@ From here, go to `Configuring Joinmarket` below.
  3. When asked, choose to restart.
 
 ##### Install Ubuntu from the Microsoft Store
-1. Open the `Microsoft Store`, search for `Ubuntu 18.04 LTS` and click `Get`.
+1. Open the `Microsoft Store`, search for `Ubuntu 24.04.1 LTS` and click `Get`.
```

Comment:
Will try testing with Ubuntu 24 under WSL2, have done that only with Ubuntu 22.

Reply:
FYI, I followed the steps with 24.04.1 recently and everything worked.

Reply:
Seems to be working, yes.